R/render_executive_summary.R

```diff
@@ -40,6 +40,7 @@ render_executive_summary <- function(data,
     params = list(
       survey_dir = survey_dir,
       real_estate_dir = real_estate_dir,
+      real_estate_flag = real_estate_flag,
```

Comment:
where does `real_estate_flag` come from?

Reply:
it's the same as is used in `create_interactive_report`, so it already exists in GlobalEnv in web_tool_script_3.

Reply:
ok.... that seems super shady, but let's try it